liquibase-integration-tests/src/test/java/liquibase/statementexecute/AddUniqueConstraintExecutorTest.java

```diff
@@ -158,7 +159,7 @@ public void execute_withSchema() throws Exception {
 
         assertCorrect("alter table [lbschem2].[adduqtest] add constraint [uq_test] unique ([coltomakeuq])", HsqlDatabase.class);
         assertCorrect("alter table \"lbcat2\".[adduqtest] add constraint [uq_test] unique ([coltomakeuq])", DB2Database.class, Db2zDatabase.class);
-        assertCorrect("alter table [lbschem2].[adduqtest] add constraint [uq_test] unique ([coltomakeuq])", H2Database.class);
+        assertCorrect("alter table [lbschem2].[adduqtest] add constraint [uq_test] unique ([coltomakeuq])", H2Database.class, BigqueryDatabase.class);
```

Comment:
unique constraints are not supported by bigQuery

Reply:
That's true! hm.. but when using the uniqueCosntraint tag the sql is being generated by core. So maybe we should we add a BigqueryUniqueConstraintGenerator that throws an exception?

Reply:
Great, the problem is the test. I added a comment explaining it.